eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -158,7 +158,7 @@ protected function configure()
                     - Refresh (add/update/remove) index on a set of content ID's:
                       <comment>ibexa:reindex --content-ids=2,34,68</comment>
                     
-                    - Refresh (add/update) index of a subtree:
+                    - Refresh (add/update) index of a subtree for a single location ID:
```

Comment:
Maybe:
```suggestion
                    - Refresh (add/update) index of a subtree for a single Location ID:
```
or `locationId`.